entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/extension/KotlinTypeExt.kt

```diff
@@ -0,0 +1,37 @@
+package godot.entrygenerator.extension
+
+import org.jetbrains.kotlin.types.KotlinType
+
+
+fun KotlinType.isCoreType(): Boolean {
```

Comment:
Have pretty much same methods in api code generation, maybe we can do common helper module for thoses later on.

Reply:
Good idea